components/Workspaces/TrackedRepoWizard/SelectOrgReposStep.tsx

```diff
@@ -78,11 +83,19 @@ export const SelectOrgReposStep = ({
 
   return (
     <div className="flex flex-col gap-4 h-96 max-h-96">
-      <div className="flex items-center gap-2 font-semibold">
-        <Avatar contributor={organization} size="xsmall" />
-        <span>{organization}</span>
+      <div className="flex w-full justify-between gap-2 font-semibold">
+        <div className="flex gap-2 justify-start align-middle">
+          <Avatar contributor={organization} size="small" />
+          <span className="flex align-middle">{organization}</span>
+        </div>
+        <div className="flex justify-end">
+          <DayRangePicker onDayRangeChanged={(e) => setRange(Number(e))} />
+        </div>
```

Comment:
refactor: no need for the div wrapper.

```suggestion
        <DayRangePicker onDayRangeChanged={(e) => setRange(Number(e))} />
```

Reply:
Thanks!